src/frontends/paddle/src/op/roi_align.cpp

```diff
@@ -0,0 +1,46 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "default_opset.hpp"
+#include "openvino/frontend/paddle/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace paddle {
+namespace op {
+NamedOutputs roi_align(const NodeContext& node) {
+    const auto data_node = node.get_input("X");
+    const auto roi_node = node.get_input("ROIs");
+
+    // limitation for batch_size = 1
```

Comment:
Why do we have this limitation? Can we support multiple batches?

Reply:
Hi, @mvafin: mainly because op input 'RoisNum' has different means between ov and paddle: for ov, it means "batch index", for paddle, it means "The number of RoIs in each image/batch", so that makes the op implement different which can't  be mappable directly. 
previously we try to use 'loop(in_current_iteration_input_idx, get_concatenated_slices), gather, broadcast' to support  multiple batches in roi_align conversion, but it looks like ov 'loop'  op has some limitations blocks this usage..
I left some comments about 'loop' op limitation in ticket [77390](https://jira.devtools.intel.com/browse/CVS-77390), I think it will be very helpful if you could give us some suggestions about this 'loop' op. thanks

Reply:
Could you add exception for batch != 0?

Reply:
Added for static cases check.